src/index.html

```diff
@@ -68,14 +66,23 @@
           <span class="auth-item__text disable-layout-shift" data-text="Register">Register</span>
         </a>
       </div>
+      <ol class="header__mobile-navigation">
```

Comment:
this is not completely correct semantically
1. if you create a list - then items of list should be list items, it means `<li><a>Link</a></li>`. You'll have only update the styles a bit.
2. It's better to use unordered list here `<ul>`